src/primitives/transaction.h

```diff
@@ -129,7 +129,12 @@ class CTxIn
 
 /** An output of a transaction.  It contains the public key that the next input
  * must be able to sign with to claim it.
+ *
+ * We align the struct to 4 bytes: that way, CTxOut will have 36 bytes: 8 for
+ * CAmount, and 28 for CScript. As a member of Coin, this leaves 4 bytes for
+ * Coin's nHeight without the need for any padding.
  */
+#pragma pack(push, 4)
```

Comment:
Does this break on oldschool 32-bit ARM where int reads are required to be aligned (and I presume this includes 64-bit ints?)

Reply:
I don't think it's a problem, we are using uint32_t everywhere and align to 4 byte, so it should be fine

Reply:
Hmm? Isn't nValue an 8-bit integer? Or am I misunerstanding what this does?

Reply:
yes its 8byte, but I thought it would still be ok because of how it's used in `Coin`. I guess I've missed something, I really need to set up travis builds... I'll close this for now until I figure out if/how I can do this correctly